app/containers/message/Markdown.js

```diff
@@ -38,14 +38,15 @@ export default class Markdown extends React.Component {
 		let m = formatText(msg);
 		m = emojify(m, { output: 'unicode' });
 		m = m.replace(/^\[([^\]]*)\]\(([^)]*)\)/, '').trim();
+		let p_edited = edited;
 		return (
 			<MarkdownRenderer
 				rules={{
 					paragraph: (node, children) => (
 						// eslint-disable-next-line
 						<Text key={node.key} style={styles.paragraph}>
 							{children}
-							{edited ? <Text style={styles.edited}> (edited)</Text> : null}
+							{p_edited ? (p_edited = false, <Text style={styles.edited}> (edited)</Text>) : null}
```

Comment:
Add translations to each file in path `app/i18n/locales`
And then in `message/Markdown`:
`import I18n from '../../i18n';`
 In Component itself:
 `{p_edited ? (p_edited = false, <Text style={styles.edited}> ({I18n.t('edited')})</Text>) : null}`

Reply:
> I just pulled the branch and it actually adds `(edited)` to the last paragraph on both iOS and Android. Check it out ;)
> ![image](https://user-images.githubusercontent.com/25078524/55977511-acc63a00-5c97-11e9-8104-f6502d26f4aa.png)

that's because ur message   is one  paragraph not   you should try  type message 

using like this  
- hello 
-  what 
-  how are you 
and test it

Reply:
yeah, I see now. Thanks for pointing it to me. But you commented the wrong comment. But you still should add translations to this PR.

Reply:
> yeah, I see now. Thanks for pointing it to me. But you commented the wrong comment. But you still should add translations to this PR.

Edited not on the translations options so I'll leave it for now